server/src/main/java/org/elasticsearch/cluster/metadata/DesiredNode.java

```diff
@@ -409,7 +346,7 @@ public String toString() {
     }
 
     public boolean hasVersion() {
-        return Strings.isNullOrBlank(version) == false;
+        return false;
     }
```

Comment:
I think it only had 2 usages (have I missed more in dependencies?) that has been removed in this pr.
Is it save to remove this method entirely?

Reply:
Thank you! Yes, it should be safe to remove the method now :+1: